crates/remote/src/ssh_session.rs

```diff
@@ -1180,6 +1180,11 @@ impl SshRemoteConnection {
 
         drop(askpass_task);
 
+        /*
+         * The master process exiting does not necessarily mean that anything is wrong,
+         * it could just be that SSH has forked into the background despite not being
+         * asked to - see https://bugzilla.mindrot.org/show_bug.cgi?id=3743
+
         if master_process.try_status()?.is_some() {
```

Comment:
Rather than comment out this code, could we keep the `if` but just have the body be a `log::info!()`, mentioning that this scenario might have happened?

I love the comment though, that's a great addition.